### PR TITLE
Create ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61126](https://cumulocity.atlassian.net/browse/MTM-61126)
Original PR: [7558](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7558)

[MTM-61126]: https://cumulocity.atlassian.net/browse/MTM-61126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix: add type validator to the alarm/event selector (#7423) [GRAFT][release/cd] (#7558)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md]: The Claude AI suggested to use the following title:
```suggestion
title: Alarm and event selector input validation added
```

File: content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61126
+version: 1021.2.1
+---
+fix: add type validator to the alarm/event selector (#7423) [GRAFT][release/cd] (#7558)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md]: The Claude AI suggested to use the following body:
```suggestion
The alarm and event selector in the Cumulocity IoT user interface previously allowed invalid input to be entered. With this fix, input validation has been added to the alarm and event selector to ensure only valid input is accepted. This change improves the user experience and prevents potential errors or unexpected behavior when using the alarm and event selector.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61126
+version: 1021.2.1
+---
+The alarm and event selector in the Cumulocity IoT user interface previously allowed invalid input to be entered. With this fix, input validation has been added to the alarm and event selector to ensure only valid input is accepted. This change improves the user experience and prevents potential errors or unexpected behavior when using the alarm and event selector.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-2-1-fix-add-type-validator-to-the-alarm-event-selector-7423-graft.md]:
```suggestion
The alarm and event selector in the {{< product-c8y-iot >}} UI previously allowed to enter invalid input. With this fix, input validation has been added to the alarm and event selector to ensure that only valid input is accepted. This change improves the user experience and prevents potential errors or unexpected behavior when using the alarm and event selector.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by DawidSAG** [comment]
@BeateRixen I think that this description is not correct, it should be something like:
```
Prevented users from adding duplicate alarm types in the alarm and event selector, ensuring each alarm type can only be selected once.
```

**Comment by BeateRixen** [comment]
> @BeateRixen I think that this description is not correct, it should be something like:
> 
> ```
> Prevented users from adding duplicate alarm types in the alarm and event selector, ensuring each alarm type can only be selected once.
> ```

@janhommes, you approved it technically :-). Please check once more.

**Pull request merged**